back/routes/experience.js

```diff
@@ -0,0 +1,8 @@
+require("dotenv/config")
+var express = require('express');
+var router = express.Router();
+const Parse = require('parse/node')
+var axios = require('axios');
+
+Parse.initialize(process.env.NODE_ENV_ID_PROJECT, process.env.NODE_ENV_PROJECT_KEY);
+Parse.serverURL = 'http://parseapi.back4app.com';
```

Comment:
We should move these into a common module and import it if it's needed in multiple places. This way we aren't initializing Parse twice, etc.

Reply:
I didn't know that was possible! That works amazing, thanks! 
That will be available in my next commit